prep_bm_host.sh

```diff
@@ -161,7 +161,7 @@ printf "\nConfiguring baremetal interface (%s) and bridge (%s)...\n\n" "$BM_INTF
 
 sudo tee "/etc/sysconfig/network-scripts/ifcfg-$BM_BRIDGE" > /dev/null << EOF
 TYPE=Bridge
-NM_CONTROLLED=no
+NM_CONTROLLED=yes
```

Comment:
What is the reason for this change for the baremetal bridge/interface?

Reply:
This is not a 4.5 specific change, I used this PR for testing with centos 8 provision host, where if NM_CONTROLLED=no, baremetal bridge won't get static IP address assigned.